src/rdf4cpp/rdf/BlankNode.cpp

```diff
@@ -1,13 +1,21 @@
 #include "BlankNode.hpp"
 
 namespace rdf4cpp::rdf {
-BlankNode::BlankNode() noexcept : Node{} {}
+BlankNode::BlankNode() noexcept : Node{NodeBackendHandle{{}, storage::node::identifier::RDFNodeType::BNode, {}}} {}
```

Comment:
Very long diff mostly not that interesting. Just wanted to show you this explicitly so you don't miss it.
I think this was buggy, at least when I compare it to `Literal` and `IRI`.

Reply:
looks good. Yes it was a bug before.